opentelemetry-api/src/opentelemetry/_events/__init__.py

```diff
@@ -0,0 +1,196 @@
+                self._name,
+                self._version,
+                self._schema_url,
+                self._attributes,
+            )
+            return self._real_event_logger
+        return self._noop_event_logger
+
+    def emit(self, event: Event) -> None:
+        self._event_logger.emit(event)
+
+
+class EventLoggerProvider(ABC):
+
+    @abstractmethod
+    def get_event_logger(
+        self,
+        name: str,
+        version: Optional[str] = None,
+        schema_url: Optional[str] = None,
+        attributes: Optional[Attributes] = None,
+    ) -> EventLogger:
+        """Returns an EventLoggerProvider for use."""
+
+
+class NoOpEventLoggerProvider(EventLoggerProvider):
+
+    def get_event_logger(
+        self,
+        name: str,
+        version: Optional[str] = None,
+        schema_url: Optional[str] = None,
+        attributes: Optional[Attributes] = None,
+    ) -> EventLogger:
+        return NoOpEventLogger(
+            name, version=version, schema_url=schema_url, attributes=attributes
+        )
+
+
+class ProxyEventLoggerProvider(EventLoggerProvider):
+
+    def get_event_logger(
+        self,
+        name: str,
+        version: Optional[str] = None,
+        schema_url: Optional[str] = None,
+        attributes: Optional[Attributes] = None,
+    ) -> EventLogger:
+        if _EVENT_LOGGER_PROVIDER:
+            return _EVENT_LOGGER_PROVIDER.get_event_logger(
+                name,
+                version=version,
+                schema_url=schema_url,
+                attributes=attributes,
+            )
+        return ProxyEventLogger(
+            name,
+            version=version,
+            schema_url=schema_url,
+            attributes=attributes,
+        )
+
+
+_EVENT_LOGGER_PROVIDER_SET_ONCE = Once()
+_EVENT_LOGGER_PROVIDER: Optional[EventLoggerProvider] = None
+_PROXY_EVENT_LOGGER_PROVIDER = ProxyEventLoggerProvider()
+
+
+def get_event_logger_provider() -> EventLoggerProvider:
+
+    global _EVENT_LOGGER_PROVIDER  # pylint: disable=global-variable-not-assigned
+    if _EVENT_LOGGER_PROVIDER is None:
+        if _OTEL_PYTHON_EVENT_LOGGER_PROVIDER not in environ:
+            return _PROXY_EVENT_LOGGER_PROVIDER
+
+        event_logger_provider: EventLoggerProvider = _load_provider(  # type: ignore
+            _OTEL_PYTHON_EVENT_LOGGER_PROVIDER, "event_logger_provider"
+        )
+
+        _set_event_logger_provider(event_logger_provider, log=False)
+
+    return cast("EventLoggerProvider", _EVENT_LOGGER_PROVIDER)
+
+
+def _set_event_logger_provider(
+    event_logger_provider: EventLoggerProvider, log: bool
+) -> None:
+    def set_elp() -> None:
+        global _EVENT_LOGGER_PROVIDER  # pylint: disable=global-statement
+        _EVENT_LOGGER_PROVIDER = event_logger_provider
+
+    did_set = _EVENT_LOGGER_PROVIDER_SET_ONCE.do_once(set_elp)
+
+    if log and did_set:
+        _logger.warning(
+            "Overriding of current EventLoggerProvider is not allowed"
+        )
+
+
```

Comment:
The logs counterparts provides also a `get_logger` helper function, should we add something similar here too?